src/nlp-checklist.html

```diff
@@ -1,165 +1,460 @@
-  <button class="checklist-item__expand" aria-label="Toggle Info" title="Toggle More Information">
-    <span class="line"></span>
-  </button>
-  <div class="info-container">
-    <div class="info">
-      Learn about various <a href="https://iq.opengenus.org/types-of-nlp-models/">types of NLP models</a> and their applications, including rule-based models, statistical models, and deep learning models.
-    </div>
-  </div>
-</li>
-
-<li class="checklist-item">
-  <input id="" type="checkbox" /><label for="" class="checkbox"></label><span class="checklist-item__title">Lexicon in NLP</span>
-  <button class="checklist-item__expand" aria-label="Toggle Info" title="Toggle More Information">
-    <span class="line"></span>
-  </button>
-  <div class="info-container">
-    <div class="info">
-      Understand the role of <a href="https://iq.opengenus.org/lexicon-in-nlp/">lexicon in NLP</a>, a collection of words and their meanings, essential for tasks like sentiment analysis and named entity recognition.
-    </div>
-  </div>
-</li>
-
-<li class="checklist-item">
-  <input id="" type="checkbox" /><label for="" class="checkbox"></label><span class="checklist-item__title">Chunking and Chinking in NLP</span>
-  <button class="checklist-item__expand" aria-label="Toggle Info" title="Toggle More Information">
-    <span class="line"></span>
-  </button>
-  <div class="info-container">
-    <div class="info">
-      Dive into the techniques of <a href="https://iq.opengenus.org/chunking-and-chinking-in-nlp/">chunking and chinking</a> in NLP, crucial for extracting meaningful phrases from sentences.
-    </div>
-  </div>
-</li>
-
-<li class="checklist-item">
-  <input id="" type="checkbox" /><label for="" class="checkbox"></label><span class="checklist-item__title">Lemmatization in NLP</span>
-  <button class="checklist-item__expand" aria-label="Toggle Info" title="Toggle More Information">
-    <span class="line"></span>
-  </button>
-  <div class="info-container">
-    <div class="info">
-      Learn about the process of <a href="https://iq.opengenus.org/lemmatization-in-nlp/">lemmatization</a> in NLP, transforming words into their base or root form for better analysis and understanding.
-    </div>
-  </div>
-</li>
-
-<li class="checklist-item">
-  <input id="" type="checkbox" /><label for="" class="checkbox"></label><span class="checklist-item__title">Stemming in NLP</span>
-  <button class="checklist-item__expand" aria-label="Toggle Info" title="Toggle More Information">
-    <span class="line"></span>
-  </button>
-  <div class="info-container">
-    <div class="info">
-      Explore the technique of <a href="https://iq.opengenus.org/stemming-in-nlp/">stemming</a> in NLP, reducing words to their base or root form by removing suffixes, useful for text normalization.
-    </div>
-  </div>
-</li>
+<div id = "contents" class="container">

```

Comment:
Standard format is:
<div class="container">

Refer Data Science checklist as a reference: https://github.com/OpenGenus/checklist/blob/main/src/data-science-checklist.html